scripts/lint.sh

```diff
@@ -1,34 +1,38 @@
 #!/usr/bin/env bash
-set -euxo pipefail
+set -euo pipefail
 
-# Check Go Formatting
+echo -n "Formatting golang code... "
```

Comment:
Just cleaning up our logging output from this lint script, so it's a little easier to follow what's going on